File: codewars/maximum_multiple/max_multiple.js
/*
	Given a divisor & a Bound. Find the largest integer N
	such that :

	N is divisible by divisor
	N <= bound
	n > 0
*/

function MaxMultiple(divisor, bound) {
	let numbers = []

	for (let i = 1; i <= bound; i++) {
		if (i % divisor === 0) {
			numbers.push(i)
		}
	}

	return numbers[numbers.length - 1]
}


console.log(MaxMultiple(2,7), "<= Got, Expected 6")

File: codewars/maximum_multiple/max_multiple.rb
	# Given a divisor & a Bound. Find the largest integer N
	# such that :

	# N is divisible by divisor
	# N <= bound
	# n > 0

def MaxMultiple divisor, bound
	numbers = []

	for i in 1..bound
		numbers << i if i % divisor == 0
	end

	return numbers.last
end

puts "#{MaxMultiple(2,7)} <= Got, Expected: 6"
puts "#{MaxMultiple(37, 200)} <= Got, Expected 185"
puts "#{MaxMultiple(7, 17)} <= Got, Expected 14"